docs/REST API/rest-api.md

```diff
@@ -5577,7 +5614,7 @@ optionally be filtered by `team_id`. If no `team_id` is specified, team profiles
 
 | Name                      | Type   | In    | Description                                                               |
 | ------------------------- | ------ | ----- | ------------------------------------------------------------------------- |
-| team_id                   | string | query | _Available in Fleet Premium_. The team ID to filter profiles.              |
+| profile_uuid | integer | body | **Required**. The UUID of the existing configuration profile you'd like to resend.|
```

Comment:
I think this change was made by mistake? The `fleet/configuration_profiles/summary` profile already exists and does expect a `team_id` query parameter.

Reply:
I think it was meant to be added to the list of parameters of "Get OS setting (configuration profile) status" below, where the `team_id` argument does not make sense (a single profile is implicitly related to a team).

Reply:
That's right! Whoops. Great catch, thank you. Updated the PR.